test/videojs-http-streaming.test.js

```diff
@@ -3718,7 +3718,7 @@ QUnit.test('switching playlists with an outstanding key request aborts request a
   );
 });
 
-QUnit.test('does not download segments if preload option set to none', function(assert) {
+QUnit.test('does not download anything until play if preload option set to none', function(assert) {
```

Comment:
This test was failing, and didn't really make sense anymore. I modified for the new behavior.